presto-product-tests/conf/docker/singlenode-ldap/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+services:
+
+  presto-master:
+    image: 'teradatalabs/centos6-java8-oracle-ldap'
+    command: /docker/volumes/presto/presto-product-tests/conf/docker/files/presto-launcher-wrapper.sh singlenode-ldap run
+    extra_hosts:
+       - "ad-test.presto.testldap.com:10.25.171.180"
```

Comment:
Please change the hostname to something that won't ever be resolved to any domain squatter's domain (testldap.com is a nice domain if you're a bad guy). Also, please replace any references to Microsoft's concrete implementation (Active Directory) with LDAP. So replace it with e.g. `ldap`.